### ISSUE TITLE
[Feature request]: delete/edit/copy a reply

### ISSUE THREAD

**Issue by Woisek** [oobabooga/text-generation-webui#4048]
What I'm currently missing are mentioned options in the title. And I don't mean the "copy last reply" option, that's not enough.
Basically I wish for those three option possibilities for any message in the chat. This would be really awesome mainly for instruct tasks.
Any chance to get this implemented? 🤔

**Comment by Ph0rk0z** [comment]
Edit is a 2 step process. You copy the reply and then replace the reply.

**Comment by Woisek** [comment]
> Edit is a 2 step process. You copy the reply and then replace the reply.

Like I said, "copy last reply" is not sufficient. And how can we "paste" something into already sent messages?

Maybe for a better understanding: basically the same functionality found in SillyTavern.

**Comment by takeraparterer** [comment]
I would like this feature too. a dedicated edit button that allows me to directly edit the reply

**Comment by Henry00IS** [comment]
Posting a comment here to re-open the issue. This is the one feature I was really looking forward to in chat mode, coming from AI Dungeon, the ability to just click on a message and change the contents. It seems silly that it's missing and this convoluted in a bunch of menu presses exclusively for the last reply. Surely it can't be that much work? Or is there a technical limitation that prevents this from working? Can we donate to make it work? 😅

**Comment by maniac-0s** [comment]
Yes this would be really helpful. If a bot response in a RP is *almost right* but not quite, one should be able to remove that part that just somehow doesn't fit.
